src/App.js

```diff
@@ -87,18 +120,12 @@ class TodoApp extends React.Component {
             todos: clearTodos,
```

Comment:
You can use `filter` without copyint items to new array.
```suggestion
            todos: this.state.todos.filter((item) => !item.completed),
```